src/components/animated/_colored-on-hover.less

```diff
@@ -0,0 +1,108 @@
+@keyframes fonnt {
+	0% {
+		color: black;
+	}
+	100% {
+		color: cornsilk;
+		text-shadow: 0px 0px 20px white;
+		box-shadow: inset 0px 0px 5px cornsilk;
+	}
+}
+@keyframes fonntrev {
```

Comment:
Name the animations a proper name something that reflects what the animation does. Also, prefix the name with `coloredOnHover` to not cause any problems with other animations.